apps/ledger-live-mobile/src/analytics/segment.ts

```diff
@@ -152,6 +159,49 @@ const getNewAddAccountsAttribues = () => {
   };
 };
 
+// This won't work with mocked accounts because they are considered as having funds even if in LLM it's displayed as 0
+const getTokensWithFunds = (accounts: Account[]): string[] => {
+  if (!accounts?.length) return [];
+
+  const uniqueTokens = new Set<string>();
+  const tokens: { ticker: string; networkName: string }[] = [];
```

Comment:
Why don't use a Map<string, elem>(); ? 

Your key can be `${token.ticker}:${token.networkName}`

With Map you can handle duplicate and you don't need to filter at the end